app/controllers/users.js

```diff
@@ -110,5 +112,27 @@ module.exports = {
         })
       });
     });
+  },
+  albumPhotosList(req, res, next) {
+    const purchase = {
+      userId: req.user.id,
+      albumId: req.params.id
+    };
+    Purchase.findOne({ where: purchase }).then(p => {
+      if (!p) {
+        next(defaultError('Album has not been purchased.'));
```

Comment:
maybe we can use a `notFoundError` instead of `defaultError`.

Also if you use `return next(...` you can avoid the else statement

Reply:
Fix in #13